pkg/kubectl/resource/builder.go

```diff
@@ -342,28 +342,29 @@ func (b *Builder) ResourceTypeOrNameArgs(allowEmptySelector bool, args ...string
 			}
 		}
 		return b
+	} else {
```

Comment:
personally i like this without the else block. putting an else block when its not necessary i feel decreases readability, but i will see what others have to say